cmd/multi_cmd.go

```diff
@@ -37,5 +37,5 @@ func multi(config *conf.GlobalConfiguration) {
 
 	models.RunHooks(bgDB, logrus.WithField("component", "hooks"))
 
-	api.ListenAndServe(l)
+	logrus.WithError(api.ListenAndServe(l)).Fatal("Error listening")
```

Comment:
This will always do a fatal exit. Which is not right. 

```
if err := api.ListenAndServe(l); err != nil {
  logrus.WithError(err).Fatal("Error listening")
}
```